t/14-isotovideo.t

```diff
@@ -102,31 +104,32 @@ subtest 'isotovideo with custom git repo parameters specified' => sub {
 subtest 'isotovideo with git refspec specified' => sub {
     chdir($pool_dir);
     unlink('vars.json') if -e 'vars.json';
-    isotovideo(opts => "casedir=$data_dir/tests test_git_refspec=deadbeef _exit_after_schedule=1");
+    combined_like { isotovideo(opts => "casedir=$data_dir/tests test_git_refspec=deadbeef _exit_after_schedule=1") } qr/Checking.*local.*deadbeef/, 'refspec picked up';
     is_in_log("Checking.*local.*deadbeef", 'refspec in local git repository would be checked out');
 };
 
 subtest 'productdir variable relative/absolute' => sub {
     chdir($pool_dir);
     unlink('vars.json') if -e 'vars.json';
-    isotovideo(opts => "casedir=$data_dir/tests _exit_after_schedule=1 productdir=$data_dir/tests");
+    combined_like { isotovideo(opts => "casedir=$data_dir/tests _exit_after_schedule=1 productdir=$data_dir/tests") } qr/scheduling shutdown/, 'shutdown scheduled';
     is_in_log('\d* scheduling.*shutdown', 'schedule has been evaluated');
     mkdir('product')                                                    unless -e 'product';
     mkdir('product/foo')                                                unless -e 'product/foo';
     symlink("$data_dir/tests/main.pm", "$pool_dir/product/foo/main.pm") unless -e "$pool_dir/product/foo/main.pm";
-    isotovideo(opts => "casedir=$data_dir/tests _exit_after_schedule=1 productdir=product/foo");
+    combined_like { isotovideo(opts => "casedir=$data_dir/tests _exit_after_schedule=1 productdir=product/foo") } qr/scheduling shutdown/, 'shutdown scheduled';
     is_in_log('\d* scheduling.*shutdown', 'schedule can still be found');
     unlink("$pool_dir/product/foo/main.pm");
     mkdir("$data_dir/tests/product") unless -e "$data_dir/tests/product";
     symlink("$data_dir/tests/main.pm", "$data_dir/tests/product/main.pm") unless -e "$data_dir/tests/product/main.pm";
-    isotovideo(opts => "casedir=$data_dir/tests _exit_after_schedule=1 productdir=product");
+    combined_like { isotovideo(opts => "casedir=$data_dir/tests _exit_after_schedule=1 productdir=product") } qr/scheduling shutdown/, 'shutdown scheduled';
     is_in_log('\d* scheduling.*shutdown', 'schedule can still be found for productdir relative to casedir');
 };
 
 subtest 'upload assets on demand even in failed jobs' => sub {
     chdir($pool_dir);
     unlink('vars.json') if -e 'vars.json';
-    isotovideo(opts => "casedir=$data_dir/tests schedule=tests/failing_module force_publish_hdd_1=foo.qcow2 qemu_no_kvm=1 arch=i386 backend=qemu qemu=i386", exit_code => 0);
+    my $module = 'tests/failing_module';
+    combined_like { isotovideo(opts => "casedir=$data_dir/tests schedule=$module force_publish_hdd_1=foo.qcow2 qemu_no_kvm=1 arch=i386 backend=qemu qemu=i386", exit_code => 0) } qr/scheduling failing_module $module\.pm/, 'module scheduled';
```

Comment:
We have a limit of 160 characters per line. I don't see tidy complaining. What's going on?

Reply:
Researching known bugs/limitations didn't yield anything helpful. `{` vs. `(sub{` makes no difference either. Even if I shorten the `opts` string the line will remain untouched... I can't figure out what actually trips it up...

Well, on the bright side, at least it doesn't complain after manually splitting it up.